transcriptic/config.py

```diff
@@ -715,11 +716,21 @@ def upload_to_uri(self, file_handle, content_type, title, name):
         key: str
             s3 key
         """
-        uri_route = self.get_route('upload_uri')
-        uri_resp = self.post(uri_route, data=json.dumps({"name": title}))
+        data = {
+            "attributes": {
+                "file_name": name,
+                "file_size": 0,
```

Comment:
Wasn't sure how to get the file_size in all cases.  It shouldn't really matter for non multipart uploading, but just a thought.

Reply:
Sounds good. Maybe add a `TODO` here for future reference?